adam/curriculum/phase1_curriculum.py

```diff
@@ -313,6 +318,88 @@ def _make_object_on_object_curriculum() -> _Phase1InstanceGroup:
     )
 
 
+def _make_object_in_other_object_curriculum() -> _Phase1InstanceGroup:
+    object_ = object_variable(
+        "object_0", INANIMATE_OBJECT, banned_properties=[IS_BODY_PART]
+    )
+    containing_object = object_variable(
+        "object_1",
+        INANIMATE_OBJECT,
+        required_properties=[HOLLOW],
+        banned_properties=[IS_BODY_PART],
+    )
+    situation_template = Phase1SituationTemplate(
+        object_variables=[object_, containing_object],
+        constraining_relations=[Relation(BIGGER_THAN, containing_object, object_)],
+        asserted_always_relations=[inside(object_, containing_object)],
+    )
+
+    return _phase1_instances(
+        "objects-in-other-objects",
+        sampled(
+            situation_template,
+            max_to_sample=100,
+            chooser=_CHOOSER,
+            ontology=GAILA_PHASE_1_ONTOLOGY,
+        ),
+    )
+
+
+def _make_people_in_cars_object_curriculum() -> _Phase1InstanceGroup:
```

Comment:
You could add a `VEHICLE` (non-perceivable) property and collapse the cars and trucks cases together